akvo/rsr/spa/app/modules/editor/section1/section1.jsx

```diff
@@ -139,8 +139,8 @@ const Info = ({ validations, fields, projectId, errors, showRequired, program, d
             />
             )}
           />
-            {!program && <ProjectPicker savedData={fields.relatedProjects[0]} projects={results && results.filter(it => it.id !== Number(projectId))} hasImportedResults={fields.hasImportedResults} {...{loading, projectId}} />}
-            {!fields.hasImportedResults && <ExternalProjects projectId={projectId} />}
+            {!program && <ProjectPicker projects={results && results.filter(it => it.id !== Number(projectId))} {...{ ...fields, loading, parentUuid }} />}
```

Comment:
Why the double spread (`...`) ? :thinking:

Reply:
the first spread is used to ```loading, parentUuid, fields``` variables to make them pass in through the component with the same variable name.
the second spread is used to extract all props in the ```fields``` variable.

Reply:
`{ ...fields, loading, parentUuid }` should be sufficient